src/util.ts

```diff
@@ -286,6 +286,9 @@ export const evalToConstant = (expr: Expr): Constant | undefined => {
     }
   } else if (isPropAccessExpr(expr)) {
     const obj = evalToConstant(expr.expr)?.constant as any;
+    if (typeof obj !== "object") {
```

Comment:
This will match null as typeof null is "object"

Reply:
I actually will remove this, we should be catching the callExpr before evaluating the property access on the string anyways. This was added because it fixed an issue caused by a bug I had (trying to eval the `split` not the value.)